packages/core/config/webpack.common.js

```diff
@@ -190,6 +191,14 @@ module.exports = (webpackEnv) => {
       new HtmlWebpackPlugin({
         title: path.basename(redwoodPaths.base),
         template: path.resolve(redwoodPaths.base, 'web/src/index.html'),
+        templateParameters: {
+          prerenderMarkup: redwoodConfig.web.experimentalPrerender
+            ? '<server-markup></server-markup>'
+            : '<!-- Redwood App Here -->', // this gets taken out by post processing anyway
+        },
+        scriptLoading: redwoodConfig.web.experimentalPrerender
```

Comment:
This bit is really crucial, it means our page can load without the JS. Afterwards the JS loads and hydrates the page.